docs/documentation/nodes.rst

```diff
@@ -0,0 +1,71 @@
+=====
+Nodes
+=====
+
+Nodes are the buildings blocks of the mathematical expressions represented by an individual.
+They are selected in the invididual's genome (see :ref:`pop_ind`).
+
+A node takes a certain number of inputs (specified by its `arity` which can also be 0) and transforms them into one output value. A priori, there are no limitations regarding the type of transformation, as long as it can be expressed in pure Python, SymPy_, PyTorch_, and NumPy_.
```

Comment:
```suggestion
A node receives data from a certain number of input nodes (specified by its `arity` which can be any non-zero number) and transforms them into an output value. A priori, there are no limitations regarding the type of transformation, as long as it can be expressed in pure Python, SymPy_, PyTorch_, and NumPy_.
```
i would avoid saying "one output" since the numpy functions operate on batches

Reply:
your change is not correct, I think, because the arity can be zero.

Reply:
good point, of course i meant "non-negative" ;)

Reply:
okay, impemented.